firebase-installations/src/androidTest/java/com/google/firebase/installations/FirebaseInstallationsInstrumentedTest.java

```diff
@@ -38,6 +38,7 @@
 import com.google.firebase.installations.remote.FirebaseInstallationServiceClient;
 import com.google.firebase.installations.remote.FirebaseInstallationServiceException;
 import com.google.firebase.installations.remote.InstallationResponse;
+import com.google.firebase.installations.remote.InstallationTokenResult;
```

Comment:
is this import necessary? I don't see it being used

Reply:
Yes, it is necessary. I moved the InstallationTokenResult file to different package. Hence, the change.

Reply:
got it